docs/window-shims-and-polyfills.md

```diff
@@ -0,0 +1,120 @@
+# Window shims and polyfills.
+
+Please consult the [gotchas](./gotchas.md), as it containes the preferred way to solve the problems where we need those shims and polyfills for.
+
+As part of the default installation we now provide a sample file on how you can provide `window` and `document` shims to your universal application. We created this on popular demand, and it serves as a starting point. 
```

Comment:
As part of the default installation we now provide a sample file on how you can provide `window` and `document` shims to your universal application. This file serves as a starting point.